python/lsst/pipe/base/pipelineIR.py

```diff
@@ -813,8 +849,19 @@ def to_file(self, filename: str):
         filename : `str`
             Location of document to write a `PipelineIR` object.
         """
-        with open(filename, 'w') as f:
-            yaml.dump(self.to_primitives(), f, sort_keys=False)
+        self.to_uri(filename)
+
+    def to_uri(self, uri: Union[ButlerURI, str]):
```

Comment:
I would rename this with "write" in the name somewhere ("write_to_uri"?); the name right now feels like it's supposed to somehow _transform_ the object into a URI and then return the URI (which obviously doesn't make sense, but it's still confusing).

Should also accept a `ButlerURI` instance as well as a string.